src/README.md

```diff
@@ -1,14 +1,14 @@
 # Celestia Specifications
 
-- [Specification](./specs/README.md)
+- [Specification](specs/specification.md)
```

Comment:
For uniformity, make links start with `./`. That's about it.

Reply:
```suggestion
- [Specification](./specs/specification.md)
```